operations/ftdc.go

```diff
@@ -81,7 +82,9 @@ func toJSON() cli.Command {
 			if err != nil {
 				return errors.Wrapf(err, "problem opening file '%s'", ftdcPath)
 			}
-			defer ftdcFile.Close()
+			defer func() {
+				_ = ftdcFile.Close()
```

Comment:
For the various places with this pattern, can we print/log the error rather than swallowing it?

Reply:
Sounds good, I will change these but leave the test file as is